gapii/cc/to_proto.h

```diff
@@ -33,34 +34,11 @@ struct ProtoConverter {
     }
 };
 
-// gapii::Slice<T> -> memory_pb::Slice*
```

Comment:
I am making some changes right now to serialize the state-block. 

This means serializing Slices. I would ask that you not remove these, since I have some changes here that will be terribly annoying to try and merge.

Reply:
Sure... I dropped the CL that removes this method... so it is back.
Note that part of the reason I removed it is that it does not even compile,
so you need to fix it if you want to use it.